# Week 5: ATAC-seq

In these 3 lectures you will learn about [chromatin accessibility](https://en.wikipedia.org/wiki/Epigenomics#Chromatin_accessibility_assays) and the [ATAC-seq protocol](https://en.wikipedia.org/wiki/ATAC-seq). We will go through a few of the first steps for analyzing ATAC-seq data, and a few types of downstream analysis.

## Overview

Each lecture will consist of approximately 50% slide-based lecture, and 50% hands-on tutorial. The slides are publicly available online, linked below each lecture, and the tutorials can be found in this repository on GitHub.

## Assignment

The [assignment](assignment.md) consists of questions for 2 tutorials: The [first_steps_tutorial.md](first_steps_tutorial.md), which is part of Lecture 1, and the [R_analysis_tutorial.md](R_analysis_tutorial.md), part of Lecture 3.

The assignment is due: Monday 04/27/2020

## Lecture 1: ATAC-seq protocol and basic processing

### Topics
- Introduction to chromatin accessibility
- Challenges of studying regulatory DNA
- Introduction to ATAC-seq protocol
- Basic steps of initial ATAC-seq analysis

### Resources
- Slides: http://databio.org/slides/open_chromatin.html
- Tutorial: [first_steps_tutorial.md](first_steps_tutorial.md)

## Lecture 2 - ATAC-seq pipeline processing

### Topics
- Pipelines
- Pipeline steps
- The problem of NuMts
- Serial alignments
- Quality control outputs
	- TSS enrichment plot
	- Fragment distribution

### Resources
- Slides: http://databio.org/slides/pipeline_pepatac.html
- Tutorial (optional): https://github.com/databio/atac_ebna2/blob/master/tutorial_basic.md

## Lecture 3 - ATAC-seq downstream analysis

### Topics
- Differential analysis
- Defining consensus peaks
- Clustering
- Footprinting
- Analysis of regions of interest

### Resources
- Slides: [Downstream ATAC analysis](http://databio.org/slides/downstream_atac_analysis.html)
- Tutorial: [R_analysis_tutorial.md](R_analysis_tutorial.md)
